ksql-cli/src/main/java/io/confluent/ksql/cli/console/Console.java

```diff
@@ -43,6 +43,7 @@
 import io.confluent.ksql.rest.entity.ArgumentInfo;
 import io.confluent.ksql.rest.entity.CommandStatusEntity;
 import io.confluent.ksql.rest.entity.ConnectorList;
+import io.confluent.ksql.rest.entity.ConnectorDescription;
```

Comment:
checkstyle failed for this due to it not being sorted lexicographically.

Reply:
Oops! Merge conflict with my other PR. Thanks for catching this :)